lib/loggly/client.js

```diff
@@ -227,7 +230,7 @@ Loggly.prototype.tagFilter = function (tags) {
     //
     // Remark: length may need to use Buffer.byteLength?
     //
-    return isSolid.test(tag) && tag.length <= 64;
+    return tag && isSolid.test(tag) && tag.length <= 64;
```

Comment:
Calculate the tag's length only when tag actually has some value otherwise return.